src/main.cpp

```diff
@@ -171,7 +151,7 @@ path_finder_test(const std::string& path, const std::string& save_path) {
     for (const terrain::Tile* tile : tile_path) {
         world.get_terrain_main()
             .get_tile(world.get_terrain_main().pos(tile))
-            ->set_material(&world.get_materials()->at(7), 5);
+            ->set_material(&world.get_materials().at(7), 5);
```

Comment:
stop using magic numbers

Reply:
This is in main. Go yell at me when it's in other files.